generators/generator-bot-adaptive/generators/app/templates/js/functions/messages/function.json

```diff
@@ -8,7 +8,8 @@
       "methods": [
         "get",
         "post"
-      ]
+      ],
+      "route": "api/messages"
```

Comment:
At least in .NET, the route is "api/{route}" since this is now driven by adapter configuration. Not sure if the same is true in the JS implementation or not.

Reply:
It is, good call. I'll resolve.